references.md

```diff
@@ -1,3 +1,4 @@
+<!-- markdown-link-check-disable -->
```

Comment:
@msevestre I don't think it's a good idea.
Better: extend the list of "is ok" status codes.
E.g. when I look on the messages from **this** link checker in the history: it seems to be always the 503 code.
So if you extend 
https://github.com/Open-Systems-Pharmacology/docs/blob/0b47c1e9de4480abd4e20cac577156c493664a23/.github/workflows/mlc_config.json#L5
with 503: should be fine (at least for **this** link checker) - and the links are still checked